project.ts

```diff
@@ -445,6 +445,14 @@ const project: CosmosProject = {
               type: "coin_received",
             }
           },
+          // --- Validator (@TODO: test it asap with @bryan)
```

Comment:
Am I understanding correctly that this means that you haven't confirmed whether this handler works yet?

I'm assuming this is needed to handle the genesis validator state? Otherwise, I think that would be the way to test it.

Reply:
Right now I implement both on the same way, genesis MsgCreateValidator by gen_txs and the same way for that handler, but I don't know how I can `test` with poktroll to generate that message on the network. 

So far it should work because is almost the same code and types. If you can help me with a `how to` to generate that msg with `poktroll` or `make` will be awesome to confirm it is working as expected.